.travis.yml

```diff
@@ -33,11 +33,15 @@ install_dl: &requirements_dl
 install_rl: &requirements_all
```

Comment:
A bit confusing why install_rl: has an annotation &requirements_all.
Feel free to refactor the way I added NLP requirements.